docs/conf.py

```diff
@@ -105,6 +105,30 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# If true, Sphinx will warn about all references where the target cannot be found.
+# Default is False. You can activate this mode temporarily using the -n command-line
+# switch.
+nitpicky = True
+
+# A list of (type, target) tuples (by default empty) that should be ignored when
+# generating warnings in “nitpicky mode”. Note that type should include the domain name
+# if present. Example entries would be ('py:func', 'int') or
+# ('envvar', 'LD_LIBRARY_PATH').
+nitpick_ignore = [
+    ("py:meth", "_open"),
+    ("py:attr", "tile"),
+    ("py:meth", "CDC.GetHandleAttrib"),
+    ("py:meth", "PIL.Image.Image.convert2byte"),
+    ("py:attr", "PIL.Image.Image.tag"),
+    ("py:attr", "PIL.Image.Image.tag_v2"),
+    ("py:attr", "PIL.Image.Image.tile"),
+    ("py:data", "PIL.Image.MAX_PIXELS"),
```

Comment:
From https://github.com/python-pillow/Pillow/issues/4775:

> The `MAX_PIXELS` reference appears to be a typo for `MAX_IMAGE_PIXELS` which is just omitted, but I can't come up with a good description for it. I'm also not sure where to put it.

Yep, that's a typo. Let's at least fix the typo here and elsewhere.

```suggestion
    ("py:data", "PIL.Image.MAX_IMAGE_PIXELS"),
```